opendevin/runtime/plugins/jupyter/setup.sh

```diff
@@ -29,8 +29,6 @@ fi
 # Install dependencies
 pip install jupyterlab notebook jupyter_kernel_gateway
 
-# Create logs directory
-sudo mkdir -p /opendevin/logs && sudo chmod 777 /opendevin/logs
```

Comment:
Why is this removed?

Reply:
He do it in the `ssh_box.py` script.

Reply:
Because agnostic image is without sudo... However, this may broke the original image logic. I don't know if the original logic will work without sudo.

Reply:
I tried it, and it didn't work, so I readded the sudo dependency. Once we completely refactor to the agnostic sandbox, we can remove sudo.